README.md

```diff
@@ -1,64 +1,47 @@
+| 8.`install_sogou.sh`    | [![job8][8]][0]  | -- | latest | [Input Method](https://pinyin.sogou.com/linux/?r=pinyin) |
+| 9.`install_vrep.sh`    | [![job9][9]][0]  | -- | v3.4.0 | [A General 3D Simulator for Robotics](http://www.coppeliarobotics.com/).|
+| 10.`install_ros.sh`    | [![job10][10]][0]  | ubuntu 16.04 | kinetics | [Robot Operating System](http://www.ros.org/) |
+| 11.`install_geographic.sh`    | [![job11][11]][0]  | -- | dev | [A Library for Solving Geodesic Problems](https://geographiclib.sourceforge.io/).|
+| 12.`install_grid_map.sh`    | [![job12][12]][0]  | ROS | 1.6.0 | [A Grid Map Library](https://github.com/anybotics/grid_map) |
+| 13.`install_ipopt.sh`    | [![job13][13]][0]  | -- | 3.12.4 | [Nonliner Programming Solver](https://projects.coin-or.org/Ipopt) |
 
-## Instruction
-* install_bit_source_list.sh   
-This script will replace `/etc/apt/sources.list` with the sources mirror of BIT, which will speed up the software installation through `apt-get`.
-
-* install_ceres.sh   
-ceres version: 1.12.0
-
-* install_cppad.sh  
-CppAD version: 20180000.0
-
-* install_ros.sh  
-This script will install the ROS according to your Ubuntu version automatically.
-
-  ubuntu 16.04 : ROS Kinetic   
-
-* install_github_lfs.sh  
-This script will install git lfs (Large File Storage), which replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com or GitHub Enterprise.
-
-* install_gtest.sh  
-This script will install Google c++ unit test lib.
-
-* install_libccd_fcl.sh  
-libccd version: 2.0  
-fcl version: 0.5.0
-
-* install_nlopt.sh  
-This script will install nlopt from source.
-
-* install_octomap.sh  
-This script will install the lastest version of [octomap](http://octomap.github.io/) from source.
-
-* install_ipopt.sh  
-Ipopt version: 3.12.4
-
-
-* install_sogou.sh  
-This script will install sougou pinyin.
-
-* install_vrep.sh  
-V-REP version: 3.4.0
-
-
-* install_latex.sh  
-This script will install TeXlive2018 and TeXStudio. It will also configure the environment of TeXStudio and install the Chinese fonts.    
-Make sure that you have replace `/etc/apt/sources.list` with the sources mirror of BIT by running `install_bit_source_list.bash` script. And there is at least 15G storage space in your /home directory.    
-Once the script has finished,restart the computer to make sure the software runs normally.     
-The BIT LaTeX template mirror is on this [website](https://coding.net/u/aRagdoll/p/LaTex-template/git). It's way more faster cloning it from the mirror than from [the original github repo](https://github.com/BIT-thesis/LaTeX-template.git).  
-
-* install_geographic.sh  
-
-
-* install_grid_map.sh  
 
 See [wiki](https://github.com/bit-ivrc/ivrcscripts/wiki) for more useful information.
+
+[0]: https://travis-ci.org/bit-ivrc/ivrcscripts

```

Comment:
Here is a PR([https://github.com/bjfish/travis-matrix-badges/pull/9#issue-198536214](https://github.com/bjfish/travis-matrix-badges/pull/9#issue-198536214)) which recommend using **travis-ci.com** as opposed to **travis-ci.org**.

Reply:
Both work and `.com` is faster in pratice.  Since it is not merged, we can not use it for now.